src/PrestaShopBundle/Controller/Admin/Improve/Design/PositionsController.php

```diff
@@ -199,4 +200,29 @@ public function unhookAction(Request $request)
             $this->generateUrl('admin_modules_positions')
         );
     }
+
+    /**
+     * Manage legacy flashes, this code must be removed
+     * when legacy edit will be migrated.
+     *
+     * @param int $conf
+     */
+    private function manageLegacyFlashes($conf)
```

Comment:
what is $conf? I'd like to see this function extracted into a meaningful service where you retrieve message if any.

Reply:
Don't really understand why the parameter have this name (in legacy), just to be consistent with what we have in url everywhere :/ 
I can add it in a specific service, but it will be removed as soon as possible.